.github/main.workflow

```diff
@@ -0,0 +1,21 @@
+workflow "build and test" {
+  on = "push"
+  resolves = "test"
+}
+
+action "prepare" {
+  uses = "docker://alpine/git"
+  args = ["submodule", "update", "--init", "--recursive"]
```

Comment:
This is pretty f-ing cool. GitHub Actions doesn't have support for submodules out of the box. Just run the alpine/git container, done!